bin/base.sh

```diff
@@ -1,6 +1,12 @@
-CH_BIN="$(cd "$(dirname "$0")" && pwd)"
+libexec_base=$(dirname "$0")
```

Comment:
Can we just use the `libexec_base` that's computed by the script?

Reply:
You mean by the script(s) including `base.sh`? Yes, that certainly works. I only wanted to be on the safe side, in case `base.sh` is ever included by anything else. But that's extremely unlikely to happen (since this "something else" in any case needs the `libexec` path to find `base.sh`). I'll drop it, then.